commit 019f7e77a341661742b179c92cbce485b7ae361e
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Wed Jan 4 23:51:32 2017 +0100

    Push KF5Threadweaver dep to the only place where it is used

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6fab26b..ace6f52 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -217,7 +217,6 @@ find_package(KF5 ${REQUIRED_KF5_VERSION} REQUIRED
         Parts
         Sonnet
         TextWidgets
-        ThreadWeaver
         Wallet
         WidgetsAddons
         WindowSystem
diff --git a/libs/widgets/tests/CMakeLists.txt b/libs/widgets/tests/CMakeLists.txt
index 3934630..744828c 100644
--- a/libs/widgets/tests/CMakeLists.txt
+++ b/libs/widgets/tests/CMakeLists.txt
@@ -4,6 +4,8 @@ include_directories(  ${KOMAIN_INCLUDES}  ${CMAKE_SOURCE_DIR}/libs/widgets)
 add_definitions(-DFILES_DATA_DIR="${CMAKE_CURRENT_SOURCE_DIR}/data/")
 add_definitions(-DFILES_OUTPUT_DIR="${CMAKE_CURRENT_BINARY_DIR}")
 
+find_package(KF5ThreadWeaver ${REQUIRED_KF5_VERSION} REQUIRED)
+
 #add_subdirectory(filedialogtester)
 
 # call: kowidgets_add_unit_test(<test-name> <sources> LINK_LIBRARIES <library> [<library> [...]] [GUI])
